1907-06-24.xml

```diff
@@ -0,0 +1,1384 @@
+                        <p>are on sale at the Company's establishment by Grand Contental Hotel, Cairo, and
+                            at Walker &amp; Meimarschi's, Alexandria.</p>
+                        <p>Purveyors to H. H. the KHEDIVE.</p>
+                        <p> Patronized by the Duke of Connaught and the Archduke Otto and all the High
+                            Life of Egypt.<measure type="indexNo">35750 18-4-80</measure></p>
+                    </div>
+                    <div type="advert" xml:id="deg-ad-sio01">
+                        <head>Sun Insurance Office,</head>
+                        <p>LONDON. Founded 1710.-Total sum insured in 1902 £487,600,000.</p>
+                        <p>Agents : LEON HELLER, Cairo, and BEHREND &amp; Co., Alexandria. 16-1-906</p>
+                    </div>
+                    <!-- start of column 3 -->
+                    <div type="advert" xml:id="deg-ad-aan03">
+                        <head>Hamburg &amp; Anglo-American Nile Co.</head>
+                        <p>Weekly departure during Winter Season by the<lb/> Luxurious First Class Tourist
+                            Steamers "VICTORIA," "PURITAN," &amp; "CERMANIA."<lb/> Regular Weekly Departures to
+                            the SECOND CATARACT by the S.S. "NUBIA."<lb/> THROUGH BOOKINGS TO KHARTOUM,
+                            GONDOKORO AND THE WHITE NILE.<lb/> Steamers and Dahabeahs for private charter.
+                            Steam Tugs and Steam Launches for hire.<lb/> FREIGHT SERVICE BY STEAM BARGES
+                            BETWEEN CAIRO AND ALEXANDRIA.<lb/> Working in conjunction and under special
+                            arrangement with the<lb/> “Upper Egypt Hotels Company."</p>
+                        <p>For details and programmes apply to HEAD OFFICE: Sharia Boulac, "Grand Continental 
+                            Hotel Buildings.” CAIRO. <measure
+                            type="indexNo">31-3-07</measure></p>
+                        <p>Hamburg-Amerika Linie.</p>
+                        <table>
+                            <row>
+                                <cell> </cell>
+                                <cell>Leaves Port Said</cell>
+                                <cell>Arrives Naples</cell>
+                                <cell>Arrives Hamburg</cell>
+                            </row>
+                            <row>
+                                <cell>S.S. Habsburg</cell>
+                                <cell><date when="1907-05-04"></date>May 4th </cell>
+                                <cell><date when="1907-05-08"></date>May 8th</cell>
+                                <cell><date when="1907-05-20"></date>May 20th </cell>
+                            </row>
+                            <row>
+                                <cell>S.S. Rheania</cell>
+                                <cell><date when="1907-06-15">June 15th</date></cell>
+                             
```

Comment:
this `</div>` tag, which closes the advertisement section, should be moved just before the `</div>` that closes the page.